docs/source/quickstart.rst

```diff
@@ -137,7 +137,7 @@ The ``HDateInfo`` object, accepts a date (either Gregorian or Hebrew), a boolean
 .. code:: python
 
     >>> from hdate import HDateInfo
-    >>> today = HDateInfo(today, diaspora=False, language="english")
+    >>> today = HDateInfo(today, location="israel", language="english")
```

Comment:
better use constants or str enums instead of strings both for location and languagge

Reply:
Nice to hear from you ☺️
Sounds like a good idea.